cbf/v1/cbf-v1.0.0.yaml

```diff
@@ -0,0 +1,591 @@
+          example: 2103N
+        location:
+          oneOf:
+            - $ref: ../../models/UnLocationLocation.yaml
+            - $ref: ../../models/FacilitySMDGLocation.yaml
+        loadLadenContainers:
+          description: Lists all the `LADEN` containers being loaded.
+          type: array
+          uniqueItems: true
+          minItems: 0
+          items:
+            $ref: '#/components/schemas/LadenContainer'
+            x-stoplight:
+              id: xwd9wgupwk8m7
+        loadEmptyContainers:
+          description: Lists all the `EMPTY` containers being loaded
+          type: array
+          minItems: 0
+          uniqueItems: true
+          items:
+            $ref: '#/components/schemas/EmptyContainer'
+            x-stoplight:
+              id: chwpisamzgh4b
+        loadBreakBulk:
+          type: array
+          minItems: 0
+          uniqueItems: true
+          items:
+            $ref: '#/components/schemas/NonContainer'
+            x-stoplight:
+              id: frcnb8jwobq7a
+        loadOutOfGauge:
+          type: array
+          minItems: 0
+          uniqueItems: true
+          items:
+            $ref: '#/components/schemas/NonContainer'
+            x-stoplight:
+              id: 02vefvs2x9qbz
+      required:
+        - location
+    LadenContainer:
+      title: LadenContainer
+      x-stoplight:
+        id: kdfqf0ojv0wqd
+      type: object
+      description: A CBF specification for a particular container type.
+      properties:
+        equipmentSizeCode:
+          type: string
+          example: '20'
+          description: 'A CBF specification for a particular equipment size and type.    '
+        averageTotalContainerWeight:
+          type: number
+          format: float
+          x-stoplight:
+            id: hubebhsjsro4k
+          description: Cargo Gross Weight plus Container Tare Weight.
+        unitsNonDG:
+          type: object
+          properties:
+            total:
+              type: integer
+              description: Total units non DG
+            totalPlugs:
+              type: integer
+              description: Subtotal units non DG with plugs (reefer)
+          required:
+            - total
+            - totalPlugs
+        unitsDG:
+          type: object
+          properties:
+            total:
+              type: integer
+              description: Total units DG
+            totalPlugs:
+              type: integer
+              description: Subtotal units DG with plugs (reefer)
+          required:
+            - total
+            - totalPlugs
+        discharge:
+          type: array
+          description: Lists all the `LADEN` containers being discharged.
+          items:
+            $ref: '#/components/schemas/DischargeLadenContainer'
+    DischargeLadenContainer:
+      title: Discharge Laden Container
+      type: object
+      x-stoplight:
+        id: eba602e7e427f
+      properties:
+        portVisitReference:
```

Comment:
I am not sure I get this structure. The `discharge` object here contains a `portVisitReference` which is specified on an array of `loadLadenContainers` which is specified next to a `portVisitReference`.
Does this mean that for every port - it is necessary to specify where the containers loaded will be discharged?

Reply:
Yes exactly.